internal/provider/resource_rediscloud_subscription.go

```diff
@@ -466,12 +466,20 @@ func resourceRedisCloudSubscriptionRead(ctx context.Context, d *schema.ResourceD
 		return diag.FromErr(err)
 	}
 
-	allowlist, err := flattenSubscriptionAllowlist(ctx, subId, api)
+	providers, err := buildCreateCloudProviders(d.Get("cloud_provider"))
 	if err != nil {
 		return diag.FromErr(err)
 	}
-	if err := d.Set("allowlist", allowlist); err != nil {
-		return diag.FromErr(err)
+
+	// CIDR whitelist is not allowed for Redis Labs internal resources subscription.
+	if redis.IntValue(providers[0].CloudAccountID) != 1 {
```

Comment:
This will be fixed in https://github.com/RedisLabs/terraform-provider-rediscloud/pull/60
The main thing is that the length of the `providers` slice needs to be checked to work during the import.

Reply:
The fix from main has been merged with the length check.